simulator-core/src/test/java/com/evbox/everon/ocpp/it/transactions/SendingTransactionRelatedMeterValues.java

```diff
@@ -0,0 +1,60 @@
+package com.evbox.everon.ocpp.it.transactions;
+
+import com.evbox.everon.ocpp.mock.StationSimulatorSetUp;
+import com.evbox.everon.ocpp.mock.csms.exchange.Authorize;
+import com.evbox.everon.ocpp.simulator.station.actions.Plug;
+import org.junit.jupiter.api.Test;
+
+import static com.evbox.everon.ocpp.mock.constants.StationConstants.*;
+import static com.evbox.everon.ocpp.mock.constants.StationConstants.DEFAULT_CONNECTOR_ID;
+import static com.evbox.everon.ocpp.mock.expect.ExpectedCount.times;
+import static com.evbox.everon.ocpp.v20.message.common.IdToken.Type.ISO_14443;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+
+public class SendingTransactionRelatedMeterValues extends StationSimulatorSetUp {
+
+    private final long METER_VALUES_INTERVAL = 1_000;
+    private final long POWER_CONSUMPTION_PER_INTERVAL = 100;
+
+    @Override
+    protected long getMeterValuesInterval() {
```

Comment:
instead of having two methods is it possible to use single getMeterValuesConfiguration and utilize MeterValuesConfiguration class here?